web/src/tippyjs.ts

```diff
@@ -193,6 +193,19 @@ export function initialize(): void {
         },
     });
 
+    tippy.delegate("body", {
+        target: ".fa-exclamation-triangle",
```

Comment:
Let's use a more specific selector here `.fa-exclamation-triangle` can be used in a lot of different places.